### PR TITLE
Add additional information in gemspec

### LINKED ISSUE #80: Specify Rails version as runtime dependency in gemspec

### PR DESCRIPTION
**Pull request by thegeorgeous**
Closes #80

### CODE REVIEW

File: invoicing.gemspec [hunk 1]
```diff
@@ -1,28 +1,36 @@
...
-DESC
+    This is a framework for generating and displaying invoices (ideal for commercial
+     Rails apps). It allows for flexible business logic; provides tools for tax
+     handling, commission calculation etc. It aims to be both developer-friendly
+     and accountant-friendly.
+  DESC
 
-  s.files = Dir["{lib}/**/*", "LICENSE", "Rakefile", "README.md"]
+  s.post_install_message = <<-POST_INSTALL
+    1.2.x will be the last version to support Ruby < 2.4 and Rails < 5.
```
Comment (sreecodeslayer) [inline invoicing.gemspec:23]: Should we start the message as a Deprecation Warning: ?

**Review by sreecodeslayer — COMMENTED** [review]
Just one comment. Rest looks good 👍

File: invoicing.gemspec (hunk 1 above)
Comment (thegeorgeous) [reply invoicing.gemspec:23]: That is generally used for classes. Maybe we should add it for FindSubclasses instead?

Comment (sreecodeslayer) [reply invoicing.gemspec:23]: Yes! that works better. Since the plan is to remove the class, it would be better to warn the usage of the class as well 👍

**Pull request closed without merging**